app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -33,11 +33,6 @@ class ParkingViewModel(
   private val _selectedParking = MutableStateFlow<Parking?>(null)
   val selectedParking: StateFlow<Parking?> = _selectedParking
 
-  // TODO: Replace with actual location
```

Comment:
Why is this removed? Is this linked to the feature you're implementing in this PR?

Reply:
Yes. I added a LaunchedEffect in the ListScreen. There's no reason between creating the viewmodel and loading parkings. The loading of the k closest parkings should be handled by the screen that uses it

Reply:
Makes sense, thanks for the explanation